tests/integration/test_utils.go

```diff
@@ -27,6 +28,28 @@ import (
 
 var app *gaiaApp.GaiaApp
 
+// Some tests require a random directory to be created when runnin IBC testing suite with gaia.
+// This is due to how CosmWasmVM initializes the VM - all IBC testing apps must have different dirs so they don't conflict.
+func GaiaAppIniterRandomDir() (ibctesting.TestingApp, map[string]json.RawMessage) {
```

Comment:
Maybe 
```suggestion
func GaiaAppIniterTempDir() (ibctesting.TestingApp, map[string]json.RawMessage) {
```
instead? The Randomness seems not the important part, it's that it's a dedicated directory

Reply:
Applied!